ampegProject/src/test/java/ChromeWebDriver.java

```diff
@@ -0,0 +1,21 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.Test;
+
+public class ChromeWebDriver implements Browser {
+  @Test
+  public void LaunchEdgeWebDriver() {
```

Comment:
ChromeWebDriver class should contain only behavior specific to Chrome.
Also, this class should not contain tests.